content/articles/implement-graphql-api-using-golang-and-mongodb-database/index.md

```diff
@@ -423,7 +428,7 @@ query Movie{
 
 Where `:id` is the `_id` value of the movie you want your GraphQL API to return.
 
-- Getting all movies
+- Getting all movies:
 
 ```go
 query Movies{
```

Comment:
Your conclusion section should include any next or final steps, summarizing the tasks the audience just completed with possible next steps.